src/components/common/filtering/ProjectsList/ProjectsList.tsx

```diff
@@ -0,0 +1,23 @@
+import { ProjectLite } from "../project";
+import ProjectListItem from "../ProjectsListItem";
+import { List } from "./StyledProjectsList";
+
+interface Props {
+  projects: ProjectLite[];
+  projectsLoaded: boolean;
+}
+
+export default function ProjectsList({ projects, projectsLoaded }: Props) {
+  if (!projectsLoaded) return <div>loading please wait</div>;
+  return (
+    <List>
+      {projects &&
+        projects.length > 0 &&
```

Comment:
Anna has made a new change to the design to include this case, please check the figma.
![image](https://user-images.githubusercontent.com/27841994/159498516-bfe18b11-78db-435c-9736-607fc6a70767.png)